From 5b4174e18042c21e6960430137e3a158ad105a4b Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Sat, 27 Sep 2014 20:48:23 +0800
Subject: [PATCH 0122/5242] MLK-9638-1 usb: test: define otg_hnp_reqd wIndex
 byte value

commit  f8d0706a71875d5925c97ea9e3dd4daff12b9046 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds otg_hnp_reqd wIndex high byte value for otg test mode.

Acked-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit c334dc80bc2672aab012cad6926d819feaf4634d)
(cherry picked from commit dd95b40fa19e522353b548554f55e8c77ac9bdfe)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/uapi/linux/usb/ch9.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/usb/ch9.h b/include/uapi/linux/usb/ch9.h
index e5414b6..f640d49 100644
--- a/include/uapi/linux/usb/ch9.h
+++ b/include/uapi/linux/usb/ch9.h
@@ -144,6 +144,7 @@
 #define	TEST_PACKET	4
 #define	TEST_FORCE_EN	5
 #define	TEST_OTG_SRP_REQD	6
+#define	TEST_OTG_HNP_REQD	7
 
 /* Status Type */
 #define USB_STATUS_TYPE_STANDARD	0
-- 
1.7.9.5

